libs/models/src/lib/user.ts

```diff
@@ -1,4 +1,8 @@
-import mongoose from "mongoose";
+import mongoose from 'mongoose';
+export interface IUser extends mongoose.Document {
```

Comment:
you are using mongoose.Document interface for model I don't think it's what you need. The build is failing for this.